6_php/activity_1.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+///exercise1
+
+$s = "This is a string";
+$n = 3.3;
+$b = true;
+$array = ["hitomi", "iroha", "akari"];
+
+echo "<br>".$s;
+echo "<br>".$n;
+echo "<br>".$b;
+echo "<br>".$array;
```

Comment:
Please change echo to print_r() so instead of 

` echo "<br>".$array ` 

it would be

`print_r($array)`